server/src/utils/MixerConnection.ts

```diff
@@ -160,18 +163,22 @@ export class MixerGenericConnection {
 
     updateFadeToBlack = () => {
         state.faders[0].fader.forEach((channel: any, index: number) => {
-            this.updateOutLevel(index)
+            this.updateOutLevel(index, -1)
         })
     }
 
-    updateOutLevels = () => {
+    updateOutLevels = (allMixersButThis: number = -1) => {

```

Comment:
You don't use the `allMixersButThis` parameter here and therefore it should be deleted